drivers/temperature/adt75/adt75.c

```diff
@@ -0,0 +1,179 @@
+/***************************************************************************//**
+ *   @file   adt75.c
+ *   @brief  Source file of ADT75 Driver.
+ *   @author Ciprian Regus (ciprian.regus@analog.com)
+********************************************************************************
+ * Copyright 2022(c) Analog Devices, Inc.
```

Comment:
even though this is an older pull request, it would be nice to change the year to 2023 in all files